seeder-start-restart.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Start-restart script for Kiwix-seeder
+#
+# Assumes an optional (yet recommended) config file in /etc/seeder.config
+# to overwrite the following variables.
+
+CONTAINER_NAME="seeder"                             # name of docker container
+IMAGE="ghcr.io/kiwix/bittorrent-seeder:latest"      # docker image to use
+
+DATA_PATH=$(pwd)/kiwix-seeder                       # path to store ZIM files (and incomplete .!qB ones) in (there's no hierarchy)
+MAX_STORAGE="10GiB"									# maximum disk-space to use
+SLEEP_INTERVAL="5m"                                 # how long to pause in-between catalog checks when using --loop
+DEBUG=""                                            # whether to print debug logs (set to 1 to enable)
+SEED_WHOLE_CATALOG=									# whether to continue if filters (or lack of) end up seeding the whole catalog (prevents accidental no-filter launch)
+
+#
+# FILTERS
+# - use glob-patterns
+# - split using pipe (|)
+# - combined with same filter
```

Comment:
This is a bit confusing. I imagine we combine with `OR` on same kind, and combine various kinds with `AND`. Probably an example would help